ads/google/imaVideo.js

```diff
@@ -769,6 +801,24 @@ export function onAdError() {
   playVideo();
 }
 
+/**
+ * Called intermittently as the ad plays, allowing us to display ad counter.
+ * @param {!Object} global
+ * @visibleForTesting
+ */
+export function onAdProgress(global) {
+  const {adPosition, totalAds} = global.getAdData();
+  const remainingTime = adsManager.getRemainingTime();
+  const remainingMinutes = Math.floor(remainingTime / 60);
+  let remainingSeconds = Math.floor(remainingTime % 60);
+  if (remainingSeconds.toString().length < 2) {
+    remainingSeconds = '0' + remainingSeconds;
+  }
+  const label = adLabel.replace('%s', adPosition).replace('%s', totalAds);
+  countdownDiv./*OK*/innerHTML
```

Comment:
@zhouyx Should it be OK to set innerHTML here?

Reply:
I am fine with using innerHTML on created div. but isn't `textContent` a better option here?